Dockerfiles/manifests/openshift/tf-cluster/main.tf

```diff
@@ -0,0 +1,28 @@
+//  Setup the core provider information.
```

Comment:
lets store the terraform state using a backend like s3
check `aws-eks/tf-cluster/main.tf`

Reply:
Added remote backend to the lupulus bucket in S3